system/Controller.php

```diff
@@ -81,9 +88,10 @@ class Controller
      */
     public function initController(RequestInterface $request, ResponseInterface $response, LoggerInterface $logger)
     {
-        $this->request  = $request;
-        $this->response = $response;
-        $this->logger   = $logger;
+        $this->request          = $request;
+        $this->response         = $response;
+        $this->logger           = $logger;
+        $this->configValidation = config(Validation::class);
```

Comment:
It is better that `$configValidation` is nullable, and if it is not null, set `Config\Validation::class` in ` setValidator()`.
Because when we don't use validation in the controller, we don't need the `Config\Validation` instance.